docs/embedding/sdk/next-js.md

```diff
@@ -116,21 +122,19 @@ export async function GET() {
 }
 ```
 
-Pass this `config` to `MetabaseProvider`
+And pass this `config` to `MetabaseProvider`
```

Comment:
makes the phrase consistent
https://github.com/metabase/metabase/blob/c47d04fd22d81ba390309bab1c0bdfb98a1cdfc6/docs/embedding/sdk/next-js.md#L182

Reply:
Fixed in 1b5d13cbe99296aa06912196dcdff338f57623e9